index.bs

```diff
@@ -1516,6 +1527,8 @@ When this operation is invoked, the authenticator must perform the following pro
     so, return an error code equivalent to "{{NotAllowedError}}" and terminate the operation.
 - If the |requireResidentKey| flag is set to |true| and the authenticator cannot store a [=Client-side-resident Credential
     Private Key=], return an error code equivalent to "{{ConstraintError}}" and terminate the operation.
+- If the |requireRateLimiting| flag is set to |true| and the authenticator cannot support [=Rate Limiting=], return an error code 
```

Comment:
@equalsJeffH 

Please feel free to create a PR suggesting as much and I will take a look.  Note that https://w3c.github.io/webauthn/#authenticator-ops already puts forward "detailed authenticator operational characteristics and requirements", but I assume a normative reference to FIDO Security requirements can be included in that section.